### DISCUSSION TITLE
ANGLE support

### DISCUSSION THREAD

**Discussion by bosoni** [raysan5/raylib#2359] [General]
Noticed that ANGLE support was long time ago, but removed.

There is raylib (4.1-dev) + ANGLE, it works D3D11, compiled 32bit lib only (and ANGLE is not the newest, last year I compiled that, but now new windows and dont have ANGLE sources on my harddrive anymore).

https://github.com/bosoni/raylib_angle_test

There are GLES*  headers, and I might modified those a little, cant remember. But raylib library is not touched so 4.0-dev -> 4.1-dev worked like a charm, just copied new files overwriting the old ones.

**Comment by raysan5** [comment]
@bosoni Oh! Very nice! Thanks for sharing!
